MiniApp/Classes/core/JavascriptBridge/MiniAppScriptMessageHandler.swift

```diff
@@ -464,6 +466,20 @@ internal class MiniAppScriptMessageHandler: NSObject, WKScriptMessageHandler {
         }
     }
 
+    func fetchPoints(callbackId: String) {
```

Comment:
all our methods signature for a callback id are like this:
```suggestion
    func fetchPoints(with callbackId: String) {
```